specs/chains/chains.spec

```diff
@@ -26,9 +26,9 @@ Level: Integration
 Type: Functional
 Importance: Critical
 Steps:
-    * Patch tekton config to sign and verify "image" with Tekton Chains
-    * Import image registry variables 
-    * Create quay secret for tekton chains
+    * Modifying the TektonConfig for Tekton Chains by storage for storing signatures: "oci"
+    * Import image registry variable 
+    * Create quay secret for Tekton Chains
```

Comment:
1. quay - it's up to the user to choose the image registry, it doesn't need to be quay 
2. it's not specific for Chains, I would go with something very generic e.g. Create secret with image registry credentials for SA 🤷🏻